src/Controller/EnMarche/ReferentController.php

```diff
@@ -144,4 +151,56 @@ public function commiteesAction(): Response
             'managedCommitteesJson' => $exporter->exportAsJson($list),
         ]);
     }
+
+    /**
+     * @Route("/organigrame", name="app_referent_organizational_chart")
+     */
+    public function organizationalChartAction(AbstractOrganizationalChartItemRepository $organizationalChartItemRepository, ReferentRepository $referentRepository)
+    {
+        /** @var Referent $referent */
+        if (!$referent = $referentRepository->findOneByEmailAndSelectPersonOrgaChart($this->getUser()->getEmailAddress())) {
+            throw $this->createAccessDeniedException('You have to be a root referent to edit this ressource');
+        }
+
+        return $this->render('referent/organizational_chart.html.twig', [
+            'organization_chart_items' => $organizationalChartItemRepository->getRootNodes(),
+            'referent' => $referent,
+        ]);
+    }
+
+    /**
+     * @Route("/organigrame/{id}", name="app_referent_referent_person_link_edit")
+     */
+    public function editReferentPersonLink(Request $request, ReferentPersonLinkRepository $referentPersonLinkRepository, ReferentRepository $referentRepository, PersonOrganizationalChartItem $personOrganizationalChartItem)
+    {
+        /** @var Referent $referent */
+        if (!$referent = $referentRepository->findOneBy(['emailAddress' => $this->getUser()->getEmailAddress()])) {
+            throw $this->createAccessDeniedException('You have to be a root referent to edit this ressource');
+        }
+
+        $form = $this->createForm(
+            ReferentPersonLinkType::class,
+            $referentPersonLinkRepository->findOrCreateByOrgaItemAndReferent($personOrganizationalChartItem, $referent)
+        );
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            /** @var ReferentPersonLink $referentPersonLink */
+            $referentPersonLink = $form->getData();
+
+            $em = $this->getDoctrine()->getManager();
+
+            $em->persist($referentPersonLink);
+            $em->flush();
+
+            $this->addFlash('success', 'Organigrame mis à jours.');
```

Comment:
`'Organigramme mis à jour.'`

Reply:
in english it's organizational chart no ?